<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>763</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1160</width>
    <height>674</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: rgb(20, 20, 20); /* 원하는 색상으로 변경 */
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="4,1">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">QLabel {
    color: rgb(166, 217, 171);
}</string>
              </property>
              <property name="text">
               <string>Research Results</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="research_txt">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QTextEdit {
	color: white;
    background-color: rgb(40, 40, 40); /* 원하는 색상으로 변경 */
	border: 1px solid rgb(255,255,255);
	border-radius: 5px;      /* (선택 사항) 모서리를 둥글게 */
}
QTextEdit:focus {
    border: 1px solid rgb(184, 247, 185);
    outline: none;           /* 기본 포커스 아웃라인 제거 */
}</string>
              </property>
              <property name="inputMethodHints">
               <set>Qt::ImhMultiLine</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,1">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <widget class="QPlainTextEdit" name="input_txt">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="focusPolicy">
               <enum>Qt::StrongFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QPlainTextEdit {
	color: white;
    background-color: rgb(40, 40, 40); /* 원하는 색상으로 변경 */
	border: 1px solid rgb(255,255,255);
	border-radius: 5px;      /* (선택 사항) 모서리를 둥글게 */
}
QPlainTextEdit:focus {
    border: 1px solid rgb(184, 247, 185);
    outline: none;           /* 기본 포커스 아웃라인 제거 */
}</string>
              </property>
              <property name="plainText">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Enter your message...</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,3,1" columnstretch="1,8,1">
              <item row="1" column="0">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>13</width>
                  <height>32</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="2" column="1">
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>65</width>
                  <height>13</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="2">
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>13</width>
                  <height>32</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="send_btn">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>1</horstretch>
                  <verstretch>1</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton {
	color: rgb(184, 247, 185);
    border: 1px solid rgb(184, 247, 185); /* 원하는 RGB 색상으로 변경 */
    border-radius: 5px;                  /* 모서리를 둥글게 (선택 사항) */
}
QPushButton:hover {
	color: rgb(184, 247, 185);
    background-color: rgb(50, 120, 50); /* 마우스 올렸을 때 더 진한 색 */
    border: 1px solid rgb(184, 247, 185); /* 테두리도 함께 진하게 (선택 사항) */
}

QPushButton:pressed {
	color: rgb(184, 247, 185);
    background-color: rgb(100, 180, 100); /* 버튼 눌렀을 때 가장 진한 색 */
    border: 1px solid rgb(184, 247, 185); /* 테두리도 함께 진하게 (선택 사항) */
}</string>
                </property>
                <property name="text">
                 <string>send</string>
                </property>
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>65</width>
                  <height>13</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="Setting">
         <attribute name="title">
          <string>setting</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="styleSheet">
               <string notr="true">QLabel {
    color: rgb(166, 217, 171);
}</string>
              </property>
              <property name="text">
               <string>Api-key</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="api_key_txt">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit {
	color: white;
    background-color: rgb(40, 40, 40); /* 원하는 색상으로 변경 */
	border: 1px solid rgb(255,255,255);
	border-radius: 5px;      /* (선택 사항) 모서리를 둥글게 */
}
QLineEdit:focus {
    border: 1px solid rgb(184, 247, 185);
    outline: none;           /* 기본 포커스 아웃라인 제거 */
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::PasswordEchoOnEdit</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <property name="placeholderText">
               <string>Enter your api key...</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,3,1">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="styleSheet">
                 <string notr="true">QLabel {
    color: rgb(166, 217, 171);
}</string>
                </property>
                <property name="text">
                 <string>temperature(0~1)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="Line" name="line_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="temp_val">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLineEdit {
	color: rgb(184, 247, 185);
    background-color: rgb(40, 40, 40); /* 원하는 색상으로 변경 */
	border: 1px solid rgb(255,255,255);
	border-radius: 5px;      /* (선택 사항) 모서리를 둥글게 */
}
QLineEdit:focus {
    border: 1px solid rgb(184, 247, 185);
    outline: none;           /* 기본 포커스 아웃라인 제거 */
}</string>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QSlider" name="temp_slider">
              <property name="styleSheet">
               <string notr="true">QSlider::handle:horizontal{
	background: rgb(166, 217, 171);
	width: 20px;
	margin: -5px 0;
	border-radius: 10px;
}
QSlider::handle:horizontal:hover{
	background: rgb(100,170,100);
}
QSlider::handle:horizontal:pressed{
	background: rgb(50,120,50);
}</string>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="pageStep">
               <number>10</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,3,1">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="styleSheet">
                 <string notr="true">QLabel {
    color: rgb(166, 217, 171);
}</string>
                </property>
                <property name="text">
                 <string>Query number(1~100)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="Line" name="line">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="query_val">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLineEdit {
	color: rgb(184, 247, 185);
    background-color: rgb(40, 40, 40); /* 원하는 색상으로 변경 */
	border: 1px solid rgb(255,255,255);
	border-radius: 5px;      /* (선택 사항) 모서리를 둥글게 */
}
QLineEdit:focus {
    border: 1px solid rgb(184, 247, 185);
    outline: none;           /* 기본 포커스 아웃라인 제거 */
}</string>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QSlider" name="query_num_slider">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">QSlider::handle:horizontal{
	background: rgb(166, 217, 171);
	width: 20px;
	margin: -5px 0;
	border-radius: 10px;
}
QSlider::handle:horizontal:hover{
	background: rgb(100,170,100);
}
QSlider::handle:horizontal:pressed{
	background: rgb(50,120,50);
}</string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,3,1">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="styleSheet">
                 <string notr="true">QLabel {
    color: rgb(166, 217, 171);
}</string>
                </property>
                <property name="text">
                 <string>url number(1~100)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="Line" name="line_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="url_val">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLineEdit {
	color: rgb(184, 247, 185);
    background-color: rgb(40, 40, 40); /* 원하는 색상으로 변경 */
	border: 1px solid rgb(255,255,255);
	border-radius: 5px;      /* (선택 사항) 모서리를 둥글게 */
}
QLineEdit:focus {
    border: 1px solid rgb(184, 247, 185);
    outline: none;           /* 기본 포커스 아웃라인 제거 */
}</string>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QSlider" name="url_num_slider">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">QSlider::handle:horizontal{
	background: rgb(166, 217, 171);
	width: 20px;
	margin: -5px 0;
	border-radius: 10px;
}
QSlider::handle:horizontal:hover{
	background: rgb(100,170,100);
}
QSlider::handle:horizontal:pressed{
	background: rgb(50,120,50);
}</string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="styleSheet">
                 <string notr="true">QLabel {
    color: rgb(166, 217, 171);
}</string>
                </property>
                <property name="text">
                 <string>Search Engine</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="selected_engine">
                <property name="styleSheet">
                 <string notr="true">QLabel {
    color: rgb(166, 217, 171);
}</string>
                </property>
                <property name="text">
                 <string>apply_Engine</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QRadioButton" name="google_btn">
              <property name="styleSheet">
               <string notr="true">QRadioButton{
	color: rgb(166, 217, 171);
}</string>
              </property>
              <property name="text">
               <string>Google</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="naver_btn">
              <property name="styleSheet">
               <string notr="true">QRadioButton{
	color: rgb(166, 217, 171);
}</string>
              </property>
              <property name="text">
               <string>Naver</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1160</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Setting">
    <property name="title">
     <string>&amp;Setting</string>
    </property>
   </widget>
   <addaction name="menu_Setting"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <tabstops>
  <tabstop>send_btn</tabstop>
  <tabstop>input_txt</tabstop>
  <tabstop>research_txt</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>send_btn</sender>
   <signal>clicked()</signal>
   <receiver>input_txt</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>794</x>
     <y>618</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>601</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
